### DISCUSSION TITLE
How to unistall

### DISCUSSION THREAD

**Discussion by BobDenny** [FZBunny/applepi#4] [General]
How do I uninstall. Raspberry Pi 4 Bullseye 64 installed the armhf. It took forever, and now it doesn't do anyuthing.

**Comment by FZBunny** [comment]
I'm sorry!  The posted ARM executable is 32-bit, and will not run on a
64-bit processor.   I will be including a 64-bit version in a few weeks.
(And explicitly stating 32 or 64 bit)
To remove the existing package, open a terminal and say:
*sudo apt-get remove applepi*
